truss/truss_config.py

```diff
@@ -267,6 +267,7 @@ class TrussConfig:
     apply_library_patches: bool = True
     # spec_version is a version string
     spec_version: str = DEFAULT_SPEC_VERSION
+    trainable: bool = False
```

Comment:
Doesn't this add an extra key that you'd end up serializing to the config?

Reply:
I don't think so. The serialization takes place in the `to_dict` method of `TrussConfig` and any keys left out aren't outputted into the .yml.